tst/launch_ci_runner.py

```diff
@@ -90,10 +90,15 @@ def run_tests_in_temp_dir(pr_number, head_repo, head_ref, commit_sha):
                 + " && python3 run_tests.py gpu.suite "
                 + "--exe "
                 + os.path.join(build_dir, "src", "artemis")
-                + " --log_file=ci_cpu_log.txt",
+                + f" --output_dir={output_dir}"
+                + " --log_file=darwin_log.txt",
             ]
             ret = subprocess.run(test_command, check=True)
 
+            # Set permissions to output files
+            subprocess.run(["chgrp", "-R", "jovian", output_dir], check=True)
+            subprocess.run(["chmod", "-R", "750", output_dir], check=True)
+
```

Comment:
If the test fails, then the permissions never get changed?

Reply:
maybe set the permissions when you create the directory, rather than after things run? https://superuser.com/questions/808345/change-default-permissions-for-new-files-and-folders

Reply:
Good catch! You're right. I think it's fine to just remove the exception and check the error code manually (as done now)? I don't think `subprocess.run` can actually crash the program to the point that we don't end up updating the github CI later on